run_dir/static/js/projects.js

```diff
@@ -48,11 +75,27 @@ function load_table() {
   
   // Display the loading spinner in the table
   $("#project_table_body").html('<tr><td colspan="'+columns.length+'" class="text-muted"><span class="glyphicon glyphicon-refresh glyphicon-spin"></span> <em>Loading..</em></td></tr>');
-  
-    old_date=$("#dateSlider").dateRangeSlider("values").min.getFullYear()+'-'+('0'+($("#dateSlider").dateRangeSlider("values").min.getMonth()+1)).slice(-2)+'-'+('0'+$("#dateSlider").dateRangeSlider("values").min.getDate()).slice(-2);
-    new_date=$("#dateSlider").dateRangeSlider("values").max.getFullYear()+'-'+('0'+($("#dateSlider").dateRangeSlider("values").max.getMonth()+1)).slice(-2)+'-'+('0'+$("#dateSlider").dateRangeSlider("values").max.getDate()).slice(-2);
-  
-  return $.getJSON("/api/v1/projects?list=" + projects_page_type+"&oldest_date="+old_date+"&youngest_date="+new_date, function(data) {
+
+    url="/api/v1/projects?list=" + projects_page_type;
+    if ($("#OpenDateSlider").dateRangeSlider("values").min !==$("#OpenDateSlider").dateRangeSlider("bounds").min ||
```

Comment:
Again, might be nice to do this with a class and then a loop to avoid multiple copies of the same code:

```javascript
$('.dateSlider').each(function(){
  var old_date = $(this).dateRangeSlider(......
  .....
});
```